SystemServices/SystemServices.cpp

```diff
@@ -500,6 +507,156 @@ namespace WPEFramework {
             returnResponse(result);
         }//end of requestSystemReboot
 
+        /*
+         * @brief This function delays the reboot in seconds.
+         * This will internally sets the tr181 fwDelayReboot parameter.
+         * @param1[in]: {"jsonrpc":"2.0","id":"3","method":"org.rdk.System.2.fwDelayReboot",
+         *                  "params":{"fwDelayReboot": int seconds}}''
+         * @param2[out]: {"jsonrpc":"2.0","id":3,"result":{"success":<bool>}}
+         * @return: Core::<StatusCode>
+         */
+
+        uint32_t SystemServices::fwDelayReboot(const JsonObject& parameters,
```

Comment:
change method name to setFirmwareRebootDelay

Reply:
modified.